api/src/main/java/org/apache/iceberg/io/FileAppender.java

```diff
@@ -48,8 +48,9 @@ default void addAll(Iterable<D> values) {
   long length();
 
   /**
-   * @return a list of offsets for file blocks if applicable, null otherwise. When available, this
+   * @return a list of offsets for file blocks, if applicable, null otherwise. When available, this
```

Comment:
I missed this earlier, but why does this say "file blocks"? This should probably be "recommended split locations".